addons/binding/org.openhab.binding.dwdunwetter/README_de.md

```diff
@@ -0,0 +1,93 @@
+
+
+
+## Channels
+
+Für jede bereitgestellte Warning werden mehrere Channels bereitgestellt. Die Channels sind jeweils durchnummeriert, Channels die mit 1 enden sind für die erste Warnung, Channels die mit 2 enden für die zweite Warnung usw. Die vom DWD gelieferten Warnungen werden dabei nach Severity (Warnstufe) sortiert und innerhalb der Warnstufe nach Beginndatum. Dadurch ist sichergestellt, dass in den Channels für die Erste Warnung (...1) immer die Warnung mit der höchsten Warnstufe steht. Werden mehr Warnungen vom DWD geliefert, als an Channels konfiguriert, werden dadurch die Warnungen mit der niedrigsten Warnstufe verworfen.
+
+| Channel | Type | Beschreibung |
+| ------- | ---- | ------------ |
+| warningN | Switch | Schalter, der auf ON steht, wenn eine Warnung vorliegt, OFF sonst. |
+| UpdatedN | Trigger Channel | Sendet das Event "NEW", wenn diese Warnung das erste mal gesendet wird. |
+| severityN | String | Warnstufe, von niedrig nach hoch: Minor, Moderate, Severe, Extreme. |
+| headlineN | String | Überschrift der Warnung, z.B. Amtliche WARNUNG vor STURMBÖEN |
+| descriptionN | String | Klartext Beschreibung der Warnung. |
+| eventN | String | Art der Warnung, z.B. STURMBÖEN |
+| effectiveN | DateTime | Zeitpunkt, an dem die Warnung ausgegeben wurde. |
+| onsetN | DateTime | Zeitpunkt, von dem an die Warnung gilt. |
+| effectiveN | DateTime | Zeitpunkt, bis zu dem die Warnung gilt. |
+| altitudeN | Number:Length | Höhe über dem Meerespiegel, ab dem die Warnung gilt. |
+| ceilingN | Number:Length | Höhe über dem Meerespiegel, bis zu dem die Warnung gilt. |
+| urgencyN | String| Zeitrahmen der Meldung, Mögliche Werte sind Future (Vorabinformation) und Immediate (Konkrete Warnung) |
+| instructionN | String | Zusatztext zur Warnung (Instruktionen und Sicherheitshinweise) |
+
+Sämtliche Channels sind ReadOnly!  
+
+Der Channel _warningN_ dient hauptsächlich dazu, um z.B. in Sitemaps dynamisch Warnungen ein- oder auszublenden, bzw. um in Regeln zu prüfen, ob überhaupt eine Warnung vorliegt. Er ist nicht geeignet um auf das Erscheinen einer Warnung zu prüfen. Denn wenn eine Warnung durch eine neue Warnung ersetzt wird, bleibt der Zustand ON, es gibt keinen Zustandswechsel. Um auf das erscheinen einer Warnung zu prüfen, sollte der Trigger-Channel _updatedN_ genutzt werden. Der feuert immer dann, wenn eine Warnung das erste mal gesendet wird. Das heißt, der feuer auch dann, wenn eine Warnung durch eine neue Warnung ersetzt wird. Weitere Erläuterungen der Channels finden sich in der Dokumentation des DWDs unter [CAP DWD Profile 1.2](https://www.dwd.de/DE/leistungen/opendata/help/warnungen/cap_dwd_profile_de_pdf.pdf?__blob=publicationFile&v=7 "CAP DWD Profile 1.2")  
+
+## Full Example
```

Comment:
There's still a lot of English in these German docs.

Reply:
I'm not sure what should be translated and what should stay english. Should I translate channel to Kanal? Is there a list of official german translations for openHab terms? 

I changed the section titles to german and fixed some spelling mistakes.

Reply:
I think it's OK this way and wouldn't translate all the openHAB concepts to German.